query/src/main/scala/filodb/query/exec/PromQlRemoteExec.scala

```diff
@@ -89,16 +92,24 @@ case class PromQlRemoteExec(queryEndpoint: String,
                             params: PromQlQueryParams,
                             numberColumnRequired: Boolean = false) extends RemoteExec {
 
-  private val columns = Seq(ColumnInfo("timestamp", ColumnType.TimestampColumn),
+  override def args: String = s"${params.toString}, queryEndpoint=$queryEndpoint, " +
+    s"requestTimeoutMs=$requestTimeoutMs, limit=${queryContext.sampleLimit}, " +
+    s"numberColumnRequired=${numberColumnRequired}"
+  private val defaultColumns = Seq(ColumnInfo("timestamp", ColumnType.TimestampColumn),
     ColumnInfo(if (numberColumnRequired) "number" else "value", ColumnType.DoubleColumn))
```

Comment:
```
ColumnInfo(if (numberColumnRequired) "number" else "value", ColumnType.DoubleColumn))
```

Can be replaced by following since we ignore the column name while comparing schemas:

```
ColumnInfo("value", ColumnType.DoubleColumn))
```